docs/source/usage/python/utils.rst

```diff
@@ -0,0 +1,13 @@
+.. _usage-python-utils:
+
+Utils
+--------
+
+memory_calculator.py
+^^^^^^^^^^^^^^^^^^^
+
+.. doxygenfile:: memory_calculator.py
```

Comment:
doxygenfile? You can use regular sphinx docs here, see e.g. https://picongpu.readthedocs.io/en/0.4.2/dev/py_postprocessing.html